build-support/bin/ci.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+function clone_repo {
+  local name=$1
+  local repo=$2
+  local branch=$3
+  git clone $repo || {
+    echo "Unable to clone $name from $repo"
+    exit 1
+  }
+  cd $name || {
+    echo "Unable to get $name from $repo"
+    exit 2
+  }
+  git checkout $branch || {
+    echo "Unable to checkout branch $branch for $name from $repo"
+    exit 3
+  }
+  cd ..
+}
+
+function publish_local {
+  local name=$1
+  local target=$2
+  cd $name || {
+    echo "Unable to get $name for publish local $target"
+    exit 4
+  }
+  ../../../sbt ++$TRAVIS_SCALA_VERSION $target || {
```

Comment:
Hmm, that's very strange.  That should only happen if you're in a directory which already has an sbt-launch.jar.  Don't spend too much time on it, if you can't figure out why, just add a TODO.